package ExceptionTest;

import org.junit.Test;

public class Test1 {

	@Test(expected =  )
	public void testPrint() {
		System.out.println("Inside the message");
		TestMessage.print();
	}
}
